.yamato/config/__shared.metafile

```diff
@@ -680,9 +670,9 @@ non_project_agents:
     flavor: m1.mac
   sdet_win_large:
     type: Unity::VM
-    image: sdet/gamecode_win10:stable
+    image: sdet/gamecode_win10:latest
```

Comment:
I think we aren't using these references anywhere atm, so this change won't affect anything, but mentioning just in case (no need to revert it if no .yml is affected)

Reply:
🎉 removed them in aa516421, no changes in the generated yaml since those were not used anywhere.
(commit is not pushed yet since "safety net pipelines" are running and don't want to cancel them 📗)